packages/api/scripts/populate.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+
+/*
+Populates database with testing logs from the testing_logs subdirectory
+*/
+
+
+
+
+const LOGS_SUBDIRECTORY = "testing_logs"
```

Comment:
You might look at using [yargs](https://github.com/yargs/yargs#simple-example) to parse command line options passed to your script:

```js
const argv = require('yargs')
   .option('collection', {
     default: 'repositories-fake',
     describe: 'collection to store test data in'
   })
   .option('repo', {
      default: 'nodejs/node',
      describe: 'repo test data is for'
    })
    .argv

const repo = encodeURIComponent (argv.repo);
const directory = argv._[0] || 'testing_logs';
```